Build failure: tplay

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

Build normally

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/297489731

### Relevant log output

```console
error[E0061]: this function takes 5 arguments but 4 arguments were supplied
    --> src/common/utils.rs:66:8
     |
66   |     if imgproc::cvt_color(&mat, &mut rgb_mat, imgproc::COLOR_BGR2RGB, 0).is_ok() {
     |        ^^^^^^^^^^^^^^^^^^----------------------------------------------- argument #5 of type `AlgorithmHint` is missing
     |
note: function defined here
    --> /build/source/target/x86_64-unknown-linux-gnu/release/build/opencv-f26f1fa25b71affe/out/opencv/imgproc.rs:6212:9
     |
6212 |     pub fn cvt_color(src: &impl ToInputArray, dst: &mut impl ToOutputArray, code: i32, dst_cn: i32, hint: core::AlgorithmHint) -> Result...
     |            ^^^^^^^^^
help: provide the argument
     |
66   -     if imgproc::cvt_color(&mat, &mut rgb_mat, imgproc::COLOR_BGR2RGB, 0).is_ok() {
66   +     if imgproc::cvt_color(&mat, &mut rgb_mat, imgproc::COLOR_BGR2RGB, 0, /* AlgorithmHint */).is_ok() {
     |

For more information about this error, try `rustc --explain E0061`.
```

### Additional context

This is due to us having a newer opencv and is fixed by bumping to 0.6.2 but that's broken due to https://github.com/maxcurzi/tplay/issues/58

### System metadata

.

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/tp/tplay/package.nix
+++ b/pkgs/by-name/tp/tplay/package.nix
@@ -4,7 +4,7 @@
   fetchFromGitHub,
   pkg-config,
   clang,
-  ffmpeg,
+  ffmpeg-headless,
   openssl,
   alsa-lib,
   opencv,
@@ -12,17 +12,17 @@
 }:
 rustPlatform.buildRustPackage rec {
   pname = "tplay";
-  version = "0.6.0";
+  version = "0.6.3";
 
   src = fetchFromGitHub {
     owner = "maxcurzi";
     repo = "tplay";
     rev = "v${version}";
-    hash = "sha256-SRn7kg5FdSimKMFowKNUIan+MrojtNO0apeehIRTzfw=";
+    hash = "sha256-JVkezG2bs99IFOTONeZZRljjbi0EhFf+DMxcfiWI4p4=";
   };
 
   useFetchCargoVendor = true;
-  cargoHash = "sha256-SzOx9IPp+TjiJpAEX8+GhZ+UEEmqNpI67S40OiYrHfM=";
+  cargoHash = "sha256-LHRTmjAwDPMOP6YQfL01leEzqRKtteU1cnUqL6UeWKk=";
   checkFlags = [
     # requires network access
     "--skip=pipeline::image_pipeline::tests::test_process"
@@ -35,20 +35,19 @@ rustPlatform.buildRustPackage rec {
     rustPlatform.bindgenHook
     pkg-config
     clang
-    ffmpeg
     makeWrapper
   ];
 
   buildInputs = [
     openssl.dev
     alsa-lib.dev
-    ffmpeg.dev
+    ffmpeg-headless.dev
     opencv
   ];
 
   postFixup = ''
     wrapProgram $out/bin/tplay \
-      --prefix PATH : "${lib.makeBinPath [ ffmpeg ]}"
+      --prefix PATH : "${lib.makeBinPath [ ffmpeg-headless ]}"
   '';
 
   meta = {

